storage/innobase/handler/ha_innodb.cc

```diff
@@ -6199,6 +5997,7 @@ static int innobase_init_files(dict_init_mode_t dict_init_mode,
     dd_space.add_file(&dd_file);
     tablespaces->push_back(&dd_space);
 
+     // we were maybe only missing this?
```

Comment:
Is this something that requires additional attention at this stage?

Reply:
Same, leftover comment.